docs/DeveloperGuide.adoc

```diff
@@ -295,17 +294,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |teacher |add, edit, and delete questions in the word banks |make corrections on what my students are supposed to learn
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |teacher |give customised word banks and definitions |can let my students practice specific problems.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |list all my word banks |
+
+|`* * *` |user |give titles to word banks |recognise them better
+
+|`* * *` |user |delete word banks |free up some memory when I don’t need it anymore
+
+|`* * *` |user |see the content of the word bank |study beforehand/make changes
```

Comment:
May need to rephrase this use case. Can't really understand how the benefit relates to the "I want to..."